sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -0,0 +1,246 @@
+from itertools import count
+import warnings
+
+import numpy as np
+
+from ...utils import check_matplotlib_support
+from ...utils import check_array
+
+
+class PartialDependenceDisplay:
+    """Partial Dependence Plot (PDP) visualization.
+
+    If is recommended to use
+    :func:`~sklearn.inspection.plot_partial_dependence` to create a
+    :class:`~sklearn.inspection.PartialDependenceDisplay`. All parameters are
+    stored as attributes.
+
+    Read more in
+    :ref:`sphx_glr_auto_examples_plot_roc_curve_visualization_api.py` and the
+    :ref:`User Guide <visualizations>`.
+
+        .. versionadded:: 0.22
+
+    Parameters
+    ----------
+    pd_results : list of (ndarray, ndarray)
+        Results of `sklearn.inspection.partial_dependence` for ``features``.
+        Each tuple corresponds to a (averaged_predictions, grid).
+
+    features : list of {(int, ), (int, int)}
```

Comment:
I think I'd rather have `list of (int,) or list of (int, int)` but don't want to argue